internal/persistence/mysql/mysql.go

```diff
@@ -16,7 +17,11 @@ import (
 )
 
 // ErrMariaDBUnsupported is the error returned when db.DialectMariaDB is attempted to be used.
-var ErrMariaDBUnsupported = errors.New("MariaDB is currently unsupported")
+var (
+	ErrMariaDBUnsupported = errors.New("MariaDB is currently unsupported")
+
+	duplicateEntryErrorCode uint16 = 0x426
```

Comment:
nit: Can we make this a constant instead?

Reply:
+1